book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rental
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
+  end
+end
```

Comment:
- [ ]  Please consider adding a method for adding rentals inside the book class. Users should able to add a rental by referring to the book itself.
> something like
```ruby
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```